setup.cfg

```diff
@@ -105,7 +104,7 @@ html =
 xml =
     lxml>=4.6.3
 plot =
-    matplotlib>=3.3.2
+    matplotlib>=3.6.1
```

Comment:
nice - I think older versions of matplotlib were failing the CI checks, so it's good you've caught this

Reply:
No worries. Only caught it because of the script 😄